chrysom/listenerClient.go

```diff
@@ -173,9 +172,6 @@ func validateListenerConfig(config *ListenerClientConfig) error {
 	if config.Listener == nil {
 		return ErrNoListenerProvided
 	}
-	if config.Logger == nil {
-		config.Logger = sallust.Default()
-	}
```

Comment:
If not having a logger can cause the program to crash, I'd keep the default logger, but change this block to:

```golang
if config.Logger == nil {
    config.Logger = zap.NewNop()
}
```